src/subscription_manager/exceptions.py

```diff
@@ -74,7 +75,14 @@ def format_bad_ca_cert_exception(self, bad_ca_cert_error, message_template):
         return message_template % bad_ca_cert_error.cert_path
 
     def format_ssl_error(self, ssl_error, message_template):
-        return message_template % str(ssl_error)
+        # SSL error has fixed form: [SSL/BIO/...: ERRORO_ID] readable string (file.c:#line)
```

Comment:
You are right. Regular expressions are hungry. It can be solved using modification of regular expression to something like this: `output_error = re.sub('^\[[^\]]*\]|\([^\)]*\)$', '', str(ssl_error))`, but when I see it, I start to be little bit scared of this :-). Golden rule: do not use regular expression, then it becomes too complicated. Thus I will revert it.

BTW: OpenSSL is very conservative in changing error messages.